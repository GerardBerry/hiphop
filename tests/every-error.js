"use hopscript"

var hh = require("hiphop");

try {
var prg =
    <hh.reactivemachine debug name="everydelay">
      <hh.inputsignal name="I"/>
      <hh.outputsignal name="O"/>
      <hh.every immediate count=2 signal_name="I">
	<hh.emit signal_name="O"/>
      </hh.every>
    </hh.reactivemachine>;
} catch (e) {
   console.log(e.message)
}
