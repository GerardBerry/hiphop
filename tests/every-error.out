*** SYNTAX ERROR
*** [LOC DISABLED --- run hop -g to get location]
*** Can't use immediate with count expression.
